tests/include/helpers.php

```diff
@@ -31,8 +31,8 @@ function force_error() {
  * A user function has to be called to force a transaction trace. PHP 7.1
  * includes dead code elimination, which means that we have to actually do
  * something that can't be eliminated: re-setting the error reporting level to
- * what it already is will do the job.
+ * what it already is will do the job. Additionally force non-zero duration for the segment not to be dropped.
  */
 function force_transaction_trace() {
-  error_reporting(error_reporting());
+  error_reporting(error_reporting()); time_nanosleep(0, 50000); // 50usec should be enough
```

Comment:
The reason call to `time_nanosleep` is on the same line as call to `error_reporting` is to avoid updates of line numbers in code level metric attributes in many tests that expect exact line number.